PC/layout/main.py

```diff
@@ -288,18 +288,17 @@ def _compile_one_py(src, dest, name, optimize, checked=True):
         return None
 
 
```

Comment:
To trigger the retest:
```suggestion
# name argument added to address bpo-37641
```
Feel free to remove it afterwards, although it could be helpful for anyone trying to figure out the reason why it was added.